### PR TITLE
feat: add resource import functionality and scripts to bootstrap user configuration

### PR DESCRIPTION
**Pull request by fxenik**
Notion link: https://www.notion.so/rudderstacks/Terraform-Provider-ea075cc6c2f341ad900e4a66a9a0384d

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests for the code
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: cmd/generatetf/generator/generator.go [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+// Generate HCL script from a given set of API sources, destinations and connections.
+func GenerateTerraform(sources []client.Source, destinations []client.Destination, connections []client.Connection) ([]byte, error) {
+	f := hclwrite.NewEmptyFile()
+	body := f.Body()
+
+	// generate source blocks
+	sourceConfigs := configs.Sources.Entries()
+	for _, src := range sources {
+		role, cm := configMeta(sourceConfigs, src.Type)
+		if cm != nil {
```
Comment (ktgowtham) [inline cmd/generatetf/generator/generator.go:64]: should we add a log for unsupported sources?

File: cmd/generatetf/generator/generator.go [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+		for k, i := range v {
+			values[k] = ctyValue(i)
+		}
+		return cty.ObjectVal(values)
+	}
+
+	return cty.EmptyObjectVal
+}
+
+func generateConnection(connection client.Connection) (*hclwrite.Block, error) {
```
Comment (ktgowtham) [inline cmd/generatetf/generator/generator.go:259]: shall we only create for connections btw supported sources and destinations?

**Review by ktgowtham — APPROVED** [review]
Added minor comments

File: cmd/generatetf/generator/generator.go (hunk 2 above)
Comment (fxenik) [reply cmd/generatetf/generator/generator.go:259]: That is also a good idea

**Pull request merged**